canonical-web-and-design/global-nav
# Update links and remove login option

## Done

- Removed option to show login links in a separate dropdown
- Updated the main dropdown according to the [copy doc](https://docs.google.com/document/d/1FeFQOztvyxnXP4Gp5nsTe0iV7mMNtY4PSfiKxvuPgWI/edit#)
  - Some of the items now have links for logins etc, added a small function to generate a link list below an item description.

## QA

- Check out this branch
- `./run` the project
- Visit http://127.0.0.1:8300/
- See that the global nav matches the [copy doc](https://docs.google.com/document/d/1FeFQOztvyxnXP4Gp5nsTe0iV7mMNtY4PSfiKxvuPgWI/edit#)

Fixes https://github.com/canonical-web-and-design/global-nav/issues/130
